app/models/applicant.rb

```diff
@@ -27,7 +27,7 @@ class Applicant < ApplicationRecord
     closed: 4
   }
 
-  audited
+  audited on: [:update]
```

Comment:
No more audits for simply creating objects. This actually fixes the bug re: changes that aren't changes.